pkg/minikube/command/kic_runner.go

```diff
@@ -185,7 +186,11 @@ func (k *kicRunner) Copy(f assets.CopyableFile) error {
 		}
 	}
 	klog.Infof("%s (temp): %s --> %s (%d bytes)", k.ociBin, src, dst, f.GetLength())
-	tf, err := ioutil.TempFile("", "tmpf-memory-asset")
+	tmpFolder := ""
+	if isSnapBinary() {
+		tmpFolder = os.Getenv("HOME")
```

Comment:
lets use a more safe way of getting the home 
how about using  
```
home, err := os.UserHomeDir()

```

Reply:
Good idea, done